pom.xml

```diff
@@ -102,7 +102,6 @@
               <encoding>UTF-8</encoding>
               <consoleOutput>true</consoleOutput>
               <failsOnError>true</failsOnError>
-              <linkXRef>false</linkXRef>
```

Comment:
Why?

Reply:
@hoegertn - I have reverted this change.  (It was at the suggestion of my IDE.)